examples/quickstart-cpp/README.md

```diff
@@ -32,7 +32,7 @@ cmake --build build
 ## Run the `Flower SuperLink`, the two clients, and the `Flower ServerApp` in separate terminals
 
```

Comment:
Should we add a bit more of instructions. For example, saying that `SuperLink` and `ServerApp` are the same python-based components as in other example; maybe indicating that a python env needs to be created, could be done pip-installing the `pyproject.toml`; indicate supernodes run in C++.